tweaks/_variables.scss

```diff
@@ -8,6 +8,8 @@ $code-block-images: false;      // Adds the programming languages logo to code b
 $web-hide-download-button: false;// Hides the download button in the web client
 $alternative-user-popup: false; // Changes the look and feel of the user popup 
 $compact-mode-tweaks: false;    // Makes compact mode more compact, among other things
+$hide-unread-mentions-bar: false;// 'unread mentions' bar can get in the way when using horizontal server list
+$server-list-above-enable-scrollbar: false;// Adds a horizontal scrollbar to the server list
```

Comment:
Seeing as how this is only used when `$server-list-above` is set, this should probably be enabled by default, because it should probably fully disable the existing scrollbar when false.

Nitpick: should be either moved up to be under `$server-list-above`, or down to be next to `$server-list-above-below`.

Reply:
Alright, ill just enable the scrollbar when `$server-list-above` is toggled, then.